haiku-apps/helpviewer/helpviewer-1.6.1.1.recipe

```diff
@@ -0,0 +1,51 @@
+SUMMARY="A help file viewer"
+DESCRIPTION="An application for viewing help files that use a simple html like mark-up. "
```

Comment:
You could keep the full description and wrap the line with a backslash to move to a new line like:
```
DESCRIPTION="An application for viewing help files that use a simple html like \
mark-up.
Unlike html, .hlp is directed at help files not a rich web experiance."
```

Reply:
Starting a new line without the backslash with move the new line also in HaikuDeport